src/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs

```diff
@@ -78,7 +78,6 @@ public static void RemoteInvoke(string[] args)
         public static int CheckCommandLineArgs(string[] args)
         {
             string[] cmdLineArgs = Environment.GetCommandLineArgs();
-            string cmdLine = Environment.CommandLine;
```

Comment:
Is this dead test code? We should probably add some tests for this in a ns17 test project.